### ISSUE TITLE
Actions menu should be navigable by arrow keys

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/nextcloud-vue#3093]
Menus need to also be navigable via arrow keys [w3.org/WAI/ARIA/apg/example-index/menu-button/menu-button-actions.html](https://www.w3.org/WAI/ARIA/apg/example-index/menu-button/menu-button-actions.html)

**Comment by raimund-schluessler** [comment]
It is navigable in principle, but you have to press the tab button once after opening the menu (or hover it with the mouse once). Then keyboard navigation works. This is however still a bug, that is already tracked in #3053.

**Comment by juliushaertl** [comment]
@jancborchardt Any thoughts? This was feedback from the accessibility workshop (iirc more a suggestion than a requirement) but I feel it does not fully apply to the actions as we may have other elements like input boxes + buttons (see the sharing actions)  in the actions where then arrow key navigation would be strange.
